### DISCUSSION TITLE
INSPIRE Theme for "Roadside trees and pruning plan"

### DISCUSSION THREAD

**Discussion by codinachssm** [INSPIRE-MIF/helpdesk#136] [Q&A]
I am looking for which INSPIRE Theme the municipal cartography of roadside trees and pruning plan should be included in.
Any help would be appreciated. Thanks!!

**Comment by KathiSchleidt** [comment]
Good question!

- Based on `roadside`, sounds like should be connected to Transport Networks Road
- Based on `trees`, would fall under Environmental Monitoring Facilities
- Based on `pruning`, fear you'll need to extend existing INSPIRE models

Sorry if this just increases the complexity of the question! ;)

**Comment by codinachssm** [comment]
OK, Kathi. Thanks for your help!
We are including it under Environmental Monitoring Facilities theme.
Thanks!
